conf/logrotate.conf

```diff
@@ -20,6 +20,23 @@ notifempty
   endscript
 }
 
+## AUTH
+/var/log/st2/st2auth.log {
```

Comment:
Usually it's better to have two entries one for normal and one for audit file - the reason for that is that users might want to keep audit log files longer than regular log files.